app/themes/ukis/info.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This contains the metadata for the UKIS theme: http://flask-themes2.readthedocs.io/en/latest/